### PR TITLE
Work on auth routing

### PR DESCRIPTION
**Pull request by kyle-cameron-zz**
- working through auth on server-side. Middleware, including bodyparser was not working properly.

### CODE REVIEW

File: app/config.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 console.log('loading db...');
-var dbconfig = require('../env/db');
```
Comment (mcfungster) [inline app/config.js:2]: revert or remove this from your commit and resubmit your pull request

File: client/index.html [hunk 2]
```diff
@@ -13,11 +11,11 @@
     <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.4.8/angular-route.js"></script>
 
     <script src="auth/auth.js"></script>
-    <script src="./services/services.js"></script>
-    <script src="app.js"></script>
     <script src="dashboard/search.js"></script>
     <script src="profile/profile.js"></script>
     <script src="landing/landing.js"></script>
```
Comment (mcfungster) [inline client/index.html:16]: please remove this from your commit as well: angular is not being loaded if this remains

Comment (kyle-cameron-zz) [reply client/index.html:16]: What's going on with these scripts? They shouldn't be interfering with angular.

**Review by mcfungster — CHANGES_REQUESTED** [review]
please remove your changes to signin and signup function, they are working, no need to modify them at this time. also, middleware is already being mounted in server.js and does not require another file at this time please remove that from your commit as well. 

there are many unnecessary changes being made to functioning code

**Pull request closed without merging**